pkg/virt-controller/services/rendervolumes.go

```diff
@@ -281,6 +283,36 @@ func withAccessCredentials(accessCredentials []v1.AccessCredential) VolumeRender
 	}
 }
 
+func withTPM(vmi *v1.VirtualMachineInstance) VolumeRendererOption {
+	return func(renderer *VolumeRenderer) error {
+		if backendstorage.HasPersistentTPMDevice(vmi) {
+			volumeName := vmi.Name + "-tpm"
+			pvcName := backendstorage.PVCPrefix + vmi.Name
```

Comment:
Can we create a function that returns the name of the PVC for the tpm? So, you can also use it in [backend-storage](https://github.com/kubevirt/kubevirt/blob/079fc5b1c3846604252931bcdd359e3c62a746bd/pkg/backend-storage/backend-storage.go#L47)

Reply:
Will do!